src/mlpack/methods/ann/layer/elu.hpp

```diff
@@ -120,28 +209,29 @@ class ELU
 
  private:
   /**
-   * Computes the ELU function
+   * Computes the value of activation function.
    *
    * @param x Input data.
    * @return f(x).
    */
   double Fn(const double x)
   {
-    if (x < DBL_MAX)
-      return (x > 0) ? x : alpha * (std::exp(x) - 1);
+    if (x < DBL_MAX) {
```

Comment:
Can you put the opening ``{`` on a new line; see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#brace-placement for more information.